gammapy/makers/background/fov.py

```diff
@@ -31,20 +36,30 @@ class FoVBackgroundMaker(Maker):
     spectral_model : SpectralModel or str
         Reference norm spectral model to use for the `FoVBackgroundModel`, if none is defined
         on the dataset. By default, use pl-norm.
+    min_counts : float
+        Minimum number of counts required outside the exclusion region
+    min_bkg : float
```

Comment:
Maybe use explicitly `min_npred_background`, so it's compatible with the names we have for `MapDataset`